source/agora/common/Transaction.d

```diff
@@ -0,0 +1,85 @@
+/*******************************************************************************
+
+    Defines the data structure of a transaction
+
+    The current design is heavily influenced by Bitcoin: as we have an UTXO,
+    starting with a simple input/output approach seems to make the most sense.
+    Script is not implemented: instead, we currently only do a simple signature
```

Comment:
```suggestion
    Script is not implemented: instead, we currently only have a simple signature
```

Reply:
The problem isn't the `do` or `have`, it's the `a`. English is complicated, I know. And I'm evil for nitpicking here. I'll stop. :P

Reply:
Yeah `an` / `a` is one of the most common mistake I do :'(